osu.Game.Tournament/Models/TournamentBeatmap.cs

```diff
@@ -41,7 +40,7 @@ public TournamentBeatmap(APIBeatmap beatmap)
             StarRating = beatmap.StarRating;
             Metadata = beatmap.Metadata;
             Difficulty = beatmap.Difficulty;
-            Covers = beatmap.BeatmapSet.AsNonNull().Covers;
```

Comment:
`AsNonNull` usage here was incorrect (fetch logic falls back to `new APIBeatmap()` which doesn't have a `BeatmapSet`).